python/fusion_config/query_default_pipeline.json

```diff
@@ -56,7 +61,7 @@
         },
         {
           "key": "fl",
-          "value": "id title productVersion productName og_image parent_s",
+          "value": "id title productVersion productName og_image parent_s author project_label datasource_label",
```

Comment:
@yannyu why the two different "fl" settings.  The "Query Fields" stage has a "return fields" setting.  Add them there.